mon-pix/app/styles/components/_checkpoint.scss

```
@@ -59,20 +59,15 @@
 }
 
 .checkpoint__continue-button {
-  padding: 0 10px;
-  width: 300px;
-  height: 50px;
-  line-height: 50px;
   display: inline-block;
+  font-weight: 700;
```

```suggestion
  font-weight: $font-bold;
```

Ok pour moi après ce petit changement 🚀